pkg/docker/docker.go

```diff
@@ -32,7 +32,9 @@ func IsLocalDockerEngineHost(dockerHost string) bool {
 		// see:
 		// https://github.com/tilt-dev/ctlptl/issues/196
 		// https://docs.docker.com/desktop/faqs/#how-do-i-connect-to-the-remote-docker-engine-api
-		return strings.Contains(dockerHost, "/var/run/docker.sock")
+		return strings.Contains(dockerHost, "/var/run/docker.sock") ||
+			// Docker Desktop for Linux - socket is in ~/.docker/desktop/docker.sock
+			strings.HasSuffix(dockerHost, "/.docker/desktop/docker.sock")
```

Comment:
can you split out this fix into a separate PR? i think that would be worthwhile

Reply:
(i wonder if we should have a library for docker socket cap detection, like we do with https://github.com/tilt-dev/clusterid)

Reply:
Ok, so just the bits required for linux support separated from the backend socket changes and the formatting?

Reply:
ya